wsdl/ver10/schema/onvif.xsd

```diff
@@ -1952,6 +1952,221 @@ decoding .A decoder shall decode every data it receives (according to its capabi
 			<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>   <!-- first Vendor then ONVIF -->
 		</xs:sequence>
 	</xs:complexType>
+
+	<!--===============================-->
+	<!--     MulticastAudioDecoder     -->
+	<!--===============================-->
+
+	<xs:complexType name="MulticastAudioDecoderConfiguration">
+		<xs:annotation>
+			<xs:documentation>
+				Configuration for the Multicast Audio Decoder.
+			</xs:documentation>
+		</xs:annotation>
+		<xs:complexContent>
+			<xs:extension base="tt:ConfigurationEntity">
+				<xs:sequence>
+					<xs:element name="Enable" type="xs:boolean">
+						<xs:annotation>
+							<xs:documentation>
+								Indicates whether the Multicast Audio Decoder is enabled or disabled.
+							</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="AudioOutputConfigurationToken" type="tt:ReferenceToken" minOccurs="0" maxOccurs="unbounded">
+						<xs:annotation>
```

Comment:
audio output configuration can be attached to a media profile independently. Any specific reason linking AudioOutputConfigurationToken with MulticastAudioDecoderConfiguration?

Reply:
> audio output configuration can be attached to a media profile independently. Any specific reason linking AudioOutputConfigurationToken with MulticastAudioDecoderConfiguration?

This new api is not related to a profile, its made independent of the media profile.

Reply:
how to get the rtsp stream uri for the client to broadcast audio back channel data with the configured multicast ip and port?

Reply:
There is no RTSP url in this case, we can discuss this in today's telco.